### PR TITLE
fix to cpu.shares documentation by @hqhq

### PR DESCRIPTION
**Pull request by SvenDowideit**
replaces #11126

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@hqhq - what do you think?

@fredlf @jamtur01 @spf13 @moxiegirl

File: docs/man/docker-run.1.md [hunk 1]
```diff
@@ -82,24 +82,38 @@ option can be set multiple times.
...
+of 1024, the first container only gets 33% of the CPU. The remaining containers
+receive 16.5%, 16.5% and 33% of the CPU.
+
+On a multi-core system, the shares of CPU time are distributed over all CPU
+cores. Even if a container is limited to less than 100% of CPU time, it can
+use 100% of each individual CPU core.
+
+For example, consider a system with more than three cores. If you start one
+container **{C0}** with **-c=512**, and another container **{C1}** with **-c=1024**,
+three CPU-intensive processes (one in **{C0}** and two in **{C1}**) this can result
```
Comment (hqhq) [inline docs/man/docker-run.1.md]: Does this sentence make sense? Sorry if I'm wrong, English is not my first language.

File: docs/man/docker-run.1.md [hunk 2]
```diff
@@ -82,24 +82,38 @@ option can be set multiple times.
...
+in the following division of CPU shares:
+
+    PID    container	CPU	CPU share
+    100    {C0}		0	100% of CPU0
```
Comment (hqhq) [inline docs/man/docker-run.1.md]: Is this a format problem? Or just github's problem?

**Comment by hqhq** [comment]
@SvenDowideit looks good to me, a few minor issues posted.

There is another question I want to ask, we usually have several files to place the same part of doc contents, like this cpu-shares, is also mentioned in `docs/man/docker-create.1.md`, should we add them all, or we can just add a link or something so it's easier to sync the contents?

File: docs/man/docker-run.1.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/man/docker-run.1.md]: yeah, its awkward as heck.

File: docs/man/docker-run.1.md (hunk 2 above)
Comment (SvenDowideit) [inline docs/man/docker-run.1.md]: which bit?

Comment (hqhq) [inline docs/man/docker-run.1.md]: `0` should be aligned with `CPU`, and `100% of CPU0` should be aligned with `CPU share`.

Comment (SvenDowideit) [inline docs/man/docker-run.1.md]: mmm, for me, it is (here in the GH diff view)

**Comment by SvenDowideit** [comment]
@hqhq - yes, we need to work out how to link documentation that is common between `create` and `run`.

**Comment by SvenDowideit** [comment]
@fredlf @jamtur01 @spf13 @moxiegirl

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**